README.md

```diff
@@ -15,11 +15,17 @@ find discrepancies between the coverage report and your suite's behavior, please
 + `solidity-coverage` is [Solcover](https://github.com/JoinColony/solcover)
 
 ### Install
+
+Run the following command in a Truffle based application or the root directory of your solidity project.
+
 ```
 $ npm install --save-dev solidity-coverage
 ```
 
 ### Run
+
+The command will be run from within your application project, for each project, you will have to install solidity-coverage.
+
```

Comment:
I think the `Run` block might benefit from a simple clarification about needing a `contracts` folder. Would you consider revising this to something like? 
```
By default solidity-coverage expects Solidity files to be located in a `contracts` folder in the root directory of your project. 
```

Reply:
For sure.